pages/alerts.rst

```diff
@@ -284,11 +284,57 @@ Client URL
 .. image:: /images/alerts_pagerduty_notification.png
 
 
+.. _alert_notification_Slack Notification:
+
+Slack alert notification
+------------------------
+The Slack alert notification allows you to send notifications using your `Webhook URL <https://api.slack.com/messaging/webhooks>`__ to your slack workspace
+in response to the configured event definitions in your Graylog server.
+
+.. image:: /images/alerts_slack_notification.png
+
+These are the supported configuration options:
+
+Webhook URL
+    An unique URL to which you send a JSON payload with the message text and some options.
+
+Channel
+    A `channel <https://api.slack.com/methods/chat.postMessage#channels>`__ to send message to.
+
+Configuration Color
+    Highlight the custom message with this `color <https://api.slack.com/reference/surfaces/formatting#quotes>`__ .
+
+Title
+    A title for this notification.
+
+Description
+    A long description for this slack notification.
+
+Custom Message
+    A custom message to be appended below the alert title.
+
+User Name (optional)
+    Addressing a specific `user  <https://api.slack.com/reference/surfaces/formatting#mentioning-users>`__ in the slack workspace.
```

Comment:
If I'm not mistaken, this is the user that the messages in Slack will appear to come from, not a user to be mentioned or tagged in the message as the link indicates.

Reply:
![notification_username](https://user-images.githubusercontent.com/6947168/97906203-e0990280-1d08-11eb-962a-aab82806d1e1.png)

the user is mentioned as above in the picture. The notification is always within the context of the event definition.

Reply:
On the config page for the Slack plugin, this field is described as "User name of the sender in Slack".  I think the same applies here.